### ISSUE TITLE
"AttributeError: 'org.python.core.ReflectedArgs' object has no attribute 'data'" with Ghidra 9.2

### ISSUE THREAD

**Issue by fmagin** [VDOO-Connected-Trust/ghidra-pyi-generator#9]
Tried generating the Python stubs for 9.2 just now and got the error:

```
Traceback (most recent call last):
  File "/home/fmagin/gits/ghidra-pyi-generator/generate_ghidra_pyi.py", line 41, in <module>
    main()
  File "/home/fmagin/gits/ghidra-pyi-generator/generate_ghidra_pyi.py", line 33, in main
    pythonscript_handler.create_mock(pyi_root, my_globals)
  File "/home/fmagin/gits/ghidra-pyi-generator/pythonscript_handler.py", line 124, in create_mock
    builtins = generate_ghidra_builtins(my_globals)
  File "/home/fmagin/gits/ghidra-pyi-generator/pythonscript_handler.py", line 87, in generate_ghidra_builtins
    classes = [type_extractor.Class.from_class(cls) for cls in builtins]
  File "/home/fmagin/gits/ghidra-pyi-generator/type_extractor.py", line 283, in from_class
    method = OverloadSet.from_reflected_function(
  File "/home/fmagin/gits/ghidra-pyi-generator/type_extractor.py", line 131, in from_reflected_function
    return OverloadSet(name=reflected_function.__name__, overloads=list(_get_overloads()))
  File "/home/fmagin/gits/ghidra-pyi-generator/type_extractor.py", line 129, in _get_overloads
    yield Overload.from_reflected_args(reflected_args, docs=docs)
  File "/home/fmagin/gits/ghidra-pyi-generator/type_extractor.py", line 89, in from_reflected_args
    return_type = reflected_args.data.getReturnType()
AttributeError: 'org.python.core.ReflectedArgs' object has no attribute 'data'
```

`reflected_args` is 

```
class ghidra.program.flatapi.FlatProgramAPI, static=false, varargs=false,flags=0, public final byte ghidra.program.flatapi.FlatProgramAPI.getByte(ghidra.program.model.address.Address) throws ghidra.program.model.mem.MemoryAccessException
	(ghidra.program.model.address.Address, )
```

in this case:
![image](https://user-images.githubusercontent.com/8415354/100786994-3b5e7080-3413-11eb-95f5-f05a712706f5.png)

**Comment by s117** [comment]
After making the changes below it works for me (with Ghidra 9.2.2).

https://github.com/s117/ghidra-pyi-generator/commit/fde7e1fe3fe820e326095c1a2944556a995b552a

I hope it helps.

**Comment by fmagin** [comment]
Is there a reason that you haven't opened a PR for this? If you don't have the time, I can take care of it, and open a PR with your (verified) commit, and take care of any additional changes that might be required to pass review.

**Comment by fmagin** [comment]
Fix included in #10

**Comment by s117** [comment]
> Is there a reason that you haven't opened a PR for this? If you don't have the time, I can take care of it, and open a PR with your (verified) commit, and take care of any additional changes that might be required to pass review.

I apologize for the delay. Thank you for the effort.
